git-publish

```diff
@@ -420,6 +420,19 @@ def edit(*filenames):
     cmd.extend(filenames)
     subprocess.call(cmd)
 
+def edit_content(content, suffix):
+    fd, tmpfile = tempfile.mkstemp(suffix=suffix)
+    fd_opened = os.fdopen(fd, 'wb')
+    fd_opened.write(content)
+    fd_opened.close()
+    edit(tmpfile)
+    fd_opened = open(tmpfile, "rb")
+    new_content = fd_opened.read()
+    fd_opened.close()
+    if tmpfile:
+        os.unlink(tmpfile)
```

Comment:
Please use `try: ... finally: ...` to ensure that the temporary file is unlinked even if an exception occurs. See `tag()` for an example.